extensions/sql-migration/src/dashboard/sqlServerDashboard.ts

```diff
@@ -0,0 +1,510 @@
+		}).component();
+
+		const migrateButtonMetadata: IActionMetadata = {
+			title: loc.DASHBOARD_MIGRATE_TASK_BUTTON_TITLE,
+			description: loc.DASHBOARD_MIGRATE_TASK_BUTTON_DESCRIPTION,
+			iconPath: {
+				dark: this.asAbsolutePath('images/migration.svg'),
+				light: this.asAbsolutePath('images/migration.svg'),
+			},
+			command: 'sqlmigration.start'
+		};
+		const migrateButton = this.createTaskButton(view, migrateButtonMetadata);
+
+		const preRequisiteListHTMLTemplate =
+			`
+			<span style="font-size: 14px;">Things you need before starting migration:</span><br>
+			<ul style="padding:0 0 0 10px;font-size:12px;">
+			<li>Azure account details</li>
+			<li>Azure SQL Managed Instance or SQL Server on Azure Virtual Machine</li>
+			<li>Backup location details</li>
+			</ul>
+			`;
+
+		const preRequisiteListElement = view.modelBuilder.dom().withProps({
+			html: preRequisiteListHTMLTemplate
+		}).component();
+
+		const preRequisiteLearnMoreLink = view.modelBuilder.hyperlink().withProps({
+			label: loc.LEARN_MORE,
+			url: '',
+			CSSStyles: {
+				'padding-left': '10px'
+			}
+		}).component();
+
+		const preReqContainer = view.modelBuilder.flexContainer().withItems([
+			preRequisiteListElement
+		]).withLayout({
+			flexFlow: 'column'
+		}).component();
+
+		preReqContainer.addItem(preRequisiteLearnMoreLink, {
+			CSSStyles: {
+				'padding-left': '10px'
+			}
+		});
+
+
+		tasksContainer.addItem(migrateButton, {
+			CSSStyles: {
+				'margin-top': '20px',
+				'padding': '10px'
+			}
+		});
+		tasksContainer.addItems([preReqContainer], {
+			CSSStyles: {
+				'padding': '10px'
+			}
+		});
+
+		return tasksContainer;
+	}
+
+	private createTaskButton(view: azdata.ModelView, taskMetaData: IActionMetadata): azdata.Component {
+		const maxHeight: number = 84;
+		const maxWidth: number = 236;
+		const buttonContainer = view.modelBuilder.button().withProperties<azdata.ButtonProperties>({
+			buttonType: azdata.ButtonType.Informational,
+			description: taskMetaData.description,
+			height: maxHeight,
+			iconHeight: 32,
+			iconPath: taskMetaData.iconPath,
+			iconWidth: 32,
+			label: taskMetaData.title,
+			title: taskMetaData.title,
+			width: maxWidth,
+		}).component();
+		buttonContainer.onDidClick(async () => {
+			if (buttonContainer.enabled && taskMetaData.command) {
+				await vscode.commands.executeCommand(taskMetaData.command);
+			}
+		});
+		return view.modelBuilder.divContainer().withItems([buttonContainer]).component();
+	}
+
+	private async refreshMigrations(): Promise<void> {
+		this._migrationStatusCardsContainer.clearItems();
+		const currentConnection = (await azdata.connection.getCurrentConnection());
+		const getMigrations = MigrationLocalStorage.getMigrations(currentConnection);
+		getMigrations.forEach((migration) => {
+			const button = this._view.modelBuilder.button().withProps({
+				label: `Migration to ${migration.targetManagedInstance.name} using controller ${migration.migrationContext.name}`
```

Comment:
localize

Reply:
This needs to be replaced. It is just a placeholder for now. I will localize the contents in the next iteration.